VERSION_NUMBER

```diff
@@ -1 +1 @@
-1.9.0
+1.10.0
```

Comment:
Maybe something like 1.9.2 for VERSION_NUMBER? Typically we use 1.10.0 for the final verified package

Reply:
@jcwchen should I use the 1.9.x version only on release branch (after this is merged to `master`) and prior to generating the test wheels as the workflow document suggested or do that even for `master`?

Reply:
Updated as suggested - downgraded release version to 1.9.100 for test package.

Reply:
@rajeevsrao Formerly ONNX <= 1.8 did bump the version to the target version directly in the main branch. However, ONNX has an auto-generated weekly package from main branch now so if the release is not completed, there might be a 1.10 onnx-weekly package first before the real 1.10 release... which is quite confusing.

@askhade Do you have any preference for it (bumping to which VERSION_NUMBER)?

Reply:
I changed it to v1.9.100 to be safe.